### PR TITLE
Merge trial

### PR DESCRIPTION
**Pull request by dvoet**
this merges in all the changes in develop since the rdbms_migration branch
- [x] **Submitter**: Rebase to develop. DO NOT SQUASH
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: Make sure documentation for code is complete
- [x] **Submitter**: Review code comments; remove done TODOs, create stories for remaining TODOs
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Add description or comments on the PR explaining the hows/whys (if not obvious)
- [x] Tell  that the PR exists if he wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
- [ ] **LR**: Initial review by LR and others.
- [ ] Comment / review / update cycle:
  - Rest of team may comments on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter updates documentation as needed
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH. **Reassign to LR** for further feedback
- [ ]  sign off
- [ ] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Squash commits, rebase if necessary
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Merge to develop 
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: Check configuration files in Jenkins in case they need changes
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by MatthewBemis** [comment]
Discussed in person, but liquibase stuff needs to be updated

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
-Add the following to your main PR comment
-=========================
-```
+
+
 - [ ] **Submitter**: Rebase to develop. DO NOT SQUASH
 - [ ] **Submitter**: Make sure Swagger is updated if API changes
 - [ ] **Submitter**: Make sure documentation for code is complete
```
Comment (jmthibault79) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: - "Make sure Liquibase is updated if the DB schema changes"

**Comment by MatthewBemis** [comment]
Not necessarily in the scope of this PR, but something that we should remember to fix: the lastModified field was updated deep in the saveObject method of GraphDAO. We need a new way to update that field now that our graph code is dead.

**Comment by MatthewBemis** [comment]
Did a first pass at making sure nothing got left out during the merge process and that looked :+1:

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jmthibault79) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: (I should have added this one to mine)

**Comment by dvoet** [comment]
liquibase done

File: src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala [hunk 2]
```diff
@@ -474,27 +511,45 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: SlickDataSo
       //we can't upgrade a read lock to a write.
```
Comment (MatthewBemis) [inline src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala:511]: outdated comment

**Comment by MatthewBemis** [comment]
Code looks good :shipit:

**Pull request merged**